server/service/src/preference/upsert.rs

```diff
@@ -54,6 +60,18 @@ pub fn upsert_preferences(
                 }
             }
 
+            if let Some(input) = input_vvm_status_input {
+                for update in input.into_iter() {
+                    input_vvm_status.upsert(connection, update.value, Some(update.store_id))?;
+                }
+            }
+
+            if let Some(input) = sort_by_vvm_status_input {
+                for update in input.into_iter() {
+                    input_vvm_status.upsert(connection, update.value, Some(update.store_id))?;
```

Comment:
```suggestion
                    sort_by_vvm_status.upsert(connection, update.value, Some(update.store_id))?;
```

Haven't actually tried this but thought it might not be correct?

Reply:
Yep, thanks James, that's my mistake! Thanks for catching it.